tfx/benchmarks/big_shuffle_benchmark.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A BigShuffle workflow."""
+
+from __future__ import absolute_import
+
+import time
+import random
+import string
+import os
+import apache_beam as beam
+from google.cloud import storage
+
+from tfx.benchmarks import benchmark_base
+
+class BigShuffleBenchmarkBase(benchmark_base.BenchmarkBase):
+  """Contains standalone Beam pipeline benchmarks, not dependent on outside
+     datasets and tensorflow dependencies."""
+
+  def __init__(self, file_size=1e6, input_file="input.txt",
+               output_file="output.txt"):
+    super(BigShuffleBenchmarkBase, self).__init__()
+    self.input_file = input_file
```

Comment:
nit: for instance variables that are "private", prefix with `_` e.g. `self._input_file` as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#id50).

Reply:
Done, I went through the classes I made to check for private variables